### PR TITLE
Demo changes 2

### PR DESCRIPTION
**Pull request by jaspreet21anand**

### CODE REVIEW

File: app/assets/javascripts/request_tile.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+RequestTile.prototype.bindEvents = function() {
+  var _this = this
+  _this.$tiles.on('click', function() {
+    _this.browseToLink(this);
+  })
+}
+
+RequestTile.prototype.browseToLink = function(tile) {
+  window.location = $(tile).find('a').attr('href')
```
Comment (ShubhamGupta) [inline app/assets/javascripts/request_tile.js:13]: Target specific elements.

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -4,7 +4,11 @@ class Admin::UsersController < Admin::BaseController
   before_action :load_user, only: [:edit, :update, :destroy, :show, :toggle_status]
 
   def index
-    @users = User.where.not(id: current_admin.id).order(:name)
+    if params[:filter]
```
Comment (ShubhamGupta) [inline app/controllers/admin/users_controller.rb:7]: Try via JS.

File: app/assets/javascripts/request_tile.js (hunk 1 above)
Comment (jaspreet21anand) [inline app/assets/javascripts/request_tile.js:13]: did in FK45

**Comment by jaspreet21anand** [comment]
Please review again

**Pull request closed without merging**